### PR TITLE
Add performance tests

### PR DESCRIPTION
**Pull request by Mykhailenko**
Added TaskSchedulingTimeTestProb test to run TaskSchedulingTimeTest with different arguments.

When TaskSchedulingTimeTest runs 1000 experiments, TaskSchedulingTimeTestProb runs single experiment.

TaskSchedulingTimeTestProb will be used in scheduling-unit-performance-tests-two-repos project.

### CODE REVIEW

File: scheduler/scheduler-server/src/test/java/performancetests/metrics/TaskSchedulingTimeTest.java [hunk 1]
```diff
@@ -101,13 +101,17 @@ public void test() throws Exception {
...
         long averageTime = totalTime / numberOfExperiments;
-        LOGGER.info(makeCSVString("AverageTaskSchedulingTime",
+        LOGGER.info(makeCSVString(name(),
                                   numberOfExperiments,
                                   timeLimit,
                                   averageTime,
                                   ((averageTime < timeLimit) ? SUCCESS : FAILURE)));
     }
 
+    public String name() {
```
Comment (ShatalovYaroslav) [inline scheduler/scheduler-server/src/test/java/performancetests/metrics/TaskSchedulingTimeTest.java]: maybe getName()

**Comment by Mykhailenko** [comment]
Retest this please

**Comment by Mykhailenko** [comment]
Retest this please

**Comment by Mykhailenko** [comment]
Retest this please

**Comment by Mykhailenko** [comment]
Retest this please

**Pull request merged**